frameworks/moveos-stdlib/src/natives/moveos_stdlib/hash.rs

```diff
@@ -21,8 +22,13 @@ fn hash<H: HashFunction<DIGEST_SIZE>, const DIGEST_SIZE: usize>(
     ty_args: Vec<Type>,
     mut args: VecDeque<Value>,
 ) -> PartialVMResult<NativeResult> {
-    debug_assert!(ty_args.is_empty());
-    debug_assert!(args.len() == 1);
+    if !ty_args.is_empty() {
+        return args_count_error(gas_params.base);
+    }
+
+    if args.len() != 2 {
+        return args_count_error(gas_params.base);
+    }
```

Comment:
```
args.len() != 1
```
This code break the test

```
2024-04-15T04:49:31.044327Z ERROR rooch_relayer::actor::relayer: Relayer execute relay block(hash: 743d2520162f43eeb42052f2158d2c73a0f16e1f4b274e21c186ab167990fe68, height: 2) failed, status: ABORTED { code: 13, location: 0000000000000000000000000000000000000000000000000000000000000002::hash }
```

Reply:
All unnecessary code for argument and type argument validation has been removed.